### ISSUE TITLE
PNaCl component is not installed in Chrome

### ISSUE THREAD

**Issue by ZhijieZhang** [cypress-io/cypress#3823]
### Is this a Feature or Bug?
This issue is more like a question. I tried asking it at `https://gitter.im/cypress-io/cypress` but didn't receive any answers :(

### Current behavior:
Chrome launched by Cypress doesn't have PNaCl components installed. If I'm testing an app which is using PNaCl then in the console I will see `NativeClient: The Portable Native Client (pnacl) component is not installed. Please consult chrome://components for more information.`
Changing the URL to `chrome://components` I see `No components are installed`.

### Desired behavior:
PNaCl components should be installed. Launch Chrome without using Cypress and changing the URL to `chrome://components`. You will see a bunch of components installed including PNaCl.

### Steps to reproduce: 
- Have the following test code:
```javascript
cy.visit('https://www.pdftron.com/samples/web/samples/viewing/viewing/')
```
- In the Chrome launched by Cypress, open the console
- Should see `NativeClient: The Portable Native Client (pnacl) component is not installed. Please consult chrome://components for more information.`

### Versions
Cypress: latest
Operating System: macOS 10.13.6
Browser: Chrome

**Comment by flotwig** [comment]
PNaCl is not installed because we pass the `--disable-component-update` flag when launching Chrome:

https://github.com/cypress-io/cypress/blob/e4bb9455419897d8b2542ba45f5989b0b5116c3b/packages/server/lib/browsers/chrome.coffee#L70

You can disable the `--disable-component-update` flag by [using the Plugin API and writing plugin code that removes it](https://docs.cypress.io/api/plugins/browser-launch-api.html). Put something like this in your `cypress/plugins/index.js`:

```js
module.exports = (on, config) => {
  on('before:browser:launch', (browser, args) => {
    return args.filter((arg) => {
      return arg !== '--disable-component-update'
    })
  })
}
```

**Comment by ZhijieZhang** [comment]
Thanks so much for your quick response! This is helpful and solves my issue.

However I have one more question for you, hope you don't mind too much:
Is there anyway I can turn off/disable/uninstall PNaCl after installing it? :)

My test scenario is that if PNaCl fails to load then the app will fall back to Emscripten. That means I need PNaCl to be installed and uninstalled to achieve a better test coverage. 

What I've tried:
1. Comment out the code you posted above in `plugs/index.js`. I see `No components are installed` after changing the URL to `chrome://components` so I was thinking this will do the trick. However when I was testing the app I still see pexe module loaded which means PNaCl didn't actually get disabled.
2. I googled and found https://productforums.google.com/forum/#!msg/chrome/NkjvcPqZH5k/LLqTh_P6LHEJ. It suggests uninstalling Chrome to uninstall all the Chrome components. So I tried uninstalling Cypress and installing it back, hoping the Chrome launched by Cypress will be "fresh" and have no components such as PNaCl. As I expected, It didn't do the trick...

I can workaround this issue on my end by forcing the app to using Emscripten, however I'd like to hear your suggestion about this. Are there any arguments like `--disable-component-update` that can achieve this easily? Thanks ver much.
